packages/calcite-components/src/utils/form.tsx

```diff
@@ -286,7 +286,7 @@ function invalidHandler(event: Event) {
       // don't clear if an icon was specified by the user
       if (
         "validationIcon" in formComponent &&
-        (formComponent.validationIcon === "" || formComponent.validationIcon === true)
+        (formComponent.validationIcon === null || formComponent.validationIcon === true)
```

Comment:
I _think_ this needs to check for `undefined` as Lumina [converts `null`s in props](https://qawebgis.esri.com/components/lumina/properties#null-casting).

@benelan Can you review this change?